inc/saiport.h

```diff
@@ -1498,6 +1501,71 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_QOS_TC_AND_COLOR_TO_MPLS_EXP_MAP,
 
+    /**
+     * @brief Port Down Error Status
+     *
+     * @type sai_port_err_status_list_t
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_ERR_STATUS_LIST,
+
+    /**
+     * @brief Fabric port Attached
+     *
+     * Signifies if the fabric port is attached
+     *
+     * @type bool
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_FABRIC_ATTACHED,
+
```

Comment:
What does this attribute FABRIC_ATTACHED mean ? Does it mean fabric port is up or down ? If so how is it different from oper status

Reply:
It is used to check if the fabric port is attached to a device .  You can skip checking for the fabric port status if it is not connected.